src/libstd/rt/mod.rs

```diff
@@ -47,6 +47,9 @@ pub mod args;
 mod at_exit_imp;
 mod libunwind;
 
+#[cfg(all(windows, target_arch="x86_64", target_env="gnu"))]
```

Comment:
It may be worth having this just be compiled for all platforms and it'll just end up being optimized away on most. That way we can catch breakage to it on all platforms instead of just 64-bit windows.